src/visitors/transpileCssProp.js

```diff
@@ -0,0 +1,104 @@
+// Most of this code was taken from @satya164's babel-plugin-css-prop
+// @see https://github.com/satya164/babel-plugin-css-prop
+import { useCssProp } from '../utils/options'
+
+const getTag = node => {
+  if (typeof node.name === 'string') return node.name
+  if (node.type === 'JSXMemberExpression') {
+    return `${getTag(node.object)}.${node.property.name}`
+  }
+  return undefined
```

Comment:
Maybe throw a syntax error here instead of returning undefined so it's easier to identify where the error occurred? Something like:

```suggestion
  throw path.buildCodeFrameError('Failed to get the name');
```